core/src/main/java/org/wso2/msf4j/internal/MicroservicesRegistryImpl.java

```diff
@@ -78,13 +78,19 @@ public void addService(Object... service) {
             services.put(svc.getClass().getAnnotation(Path.class).value(), svc);
         }
         updateMetadata();
-        Arrays.stream(service).forEach(svc -> log.info("Added microservice: " + svc));
+        Arrays.stream(service).forEach(svc -> {
+            if (log.isDebugEnabled()) {
+                log.debug("Added microservice: " + svc);
+            }
+        });
     }
 
     public void addService(String basePath, Object service) {
         services.put(basePath, service);
         metadata.addMicroserviceMetadata(service, basePath);
-        log.info("Added microservice: " + service);
+        if (log.isDebugEnabled()) {
+            log.debug("Added microservice: " + service);
```

Comment:
No need to check `log.isDebugEnabled()` here. SLF4J internally checks that. 
Also lets use parameterized log message `log.debug("Added microservice {}.", service); 
refer: https://www.slf4j.org/faq.html#logging_performance